extension/bootstrap.js

```diff
@@ -78,20 +78,23 @@ var PersonasPlusBootstrapAddon = {
         Cu.unload("resource://personas/modules/personas.js");
         this.requestRemovePrerequisites(data);
         if (reason == ADDON_DISABLE || reason == ADDON_UNINSTALL) {
-            // modules that come with Firefox
-            Cu.import("resource://gre/modules/XPCOMUtils.jsm");
-            // LightweightThemeManager may not be not available (Firefox < 3.6 or Thunderbird)
-            try {
-                Cu.import("resource://gre/modules/LightweightThemeManager.jsm");
-            } catch (e) {
-                LightweightThemeManager = null;
-            }
-            try {
-                LightweightThemeManager.forgetUsedTheme(LightweightThemeManager.currentTheme.id);
-            } catch (e) {
+            var prompt = Components.classes["@mozilla.org/embedcomp/prompt-service;1"].getService(Components.interfaces.nsIPromptService);
+            if (prompt.confirm(null, "Personas Plus", "Would you like to remove current theme?")) {
```

Comment:
But when add-on is disabled, user may want to remove Light Weight theme, so that she can use the default theme. So this prompt should be available for all cases, I suppose.

What do you think?